src/Domain/Structure/Neighbors.hpp

```diff
@@ -34,29 +35,29 @@ class Neighbors {
   /// \param orientation This OrientationMap takes objects in the logical
   /// coordinate frame of the host Element and maps them to the logical
   /// coordinate frame of the neighbor Element.
-  Neighbors(std::unordered_set<ElementId<VolumeDim>> ids,
+  Neighbors(std::unordered_set<IdType> ids,
```

Comment:
Suggestion: add a constructor with a single `IdType` value? Then you don't have to add curly braces in so many places.

Reply:
I can't think of a reason where that would subtly give unexpected results. As long as we document it, I think it'll work well :)

Reply:
done